tests/playwright/pages/wp-admin-page.ts

```diff
@@ -28,41 +30,58 @@ export default class WpAdminPage extends BasePage {
 		await this.page.waitForSelector( 'text=Dashboard' );
 	}
 
-	async openNewPage( editorType: string = '', setPageName: boolean = true ) {
-		if ( ! await this.page.$( '.e-overview__create > a' ) ) {
-			await this.gotoDashboard();
+	async openNewPage( setWithApi : boolean = true, setPageName : boolean = true ) {
+		if ( setWithApi ) {
+			await this.createNewPostWithAPI();
+		} else {
+			await this.createNewPostFromDashboard( setPageName );
 		}
 
-		await this.page.click( '.e-overview__create > a' );
 		await this.page.waitForLoadState( 'load', { timeout: 20000 } );
 		await this.waitForPanel();
-
 		await this.closeAnnouncementsIfVisible();
-		if ( setPageName !== false ) {
-			await this.setPageName( editorType );
-		}
 
 		return new EditorPage( this.page, this.testInfo );
 	}
 
-	async setPageName( editorType: string ) {
-		const selector = 'editor_v2' === editorType
-			? 'button[aria-label="Post Settings"]'
-			: '#elementor-panel-footer-settings';
+	async createNewPostWithAPI() {
+		const request: APIRequestContext = this.page.context().request,
+			postDataInitial = {
+				title: 'Playwright Test Page - Uninitialized',
+				content: '',
+			},
+			postId = await create( request, 'posts', postDataInitial ),
+			postDataUpdated = {
+				title: `Playwright Test Page #${ postId }`,
```

Comment:
I have used this title so that on localhost in the Trash it is easier to identify which posts were generated as part of the playwright tests. This will help a lot when you want to clean up the existing posts or if you would like to check the results of a Playwright test on localhost.